scopes/scope/scope/ui/components-drawer/components.drawer.tsx

```diff
@@ -17,13 +17,24 @@ import { ellipsis } from '@teambit/design.ui.styles.ellipsis';
 import { useScopeQuery } from '@teambit/scope.ui.hooks.use-scope';
 import styles from './components-drawer.module.scss';
 
+/* eslint-disable no-unused-vars */
+const ScopeTreeContext = createContext({ collapsed: true, setCollapsed: (x: boolean) => {} });
```

Comment:
better not adding eslint ignores. you can replace it with a type instead, like:
createContext<{collapsed: boolean, setCollapsed: (x:boolean) => void }>({ collapsed: true, setCollapsed: () => {} });